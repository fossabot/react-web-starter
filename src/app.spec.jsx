import App from './app'
import React from 'react'
import { shallow } from 'enzyme'

describe('App', () => {
  it('should render correctly', () => {
    const app = shallow(<App />)

    expect(app.find('h1').text()).not.toHaveLength(0)
    expect(app).toMatchSnapshot()
  })
})
