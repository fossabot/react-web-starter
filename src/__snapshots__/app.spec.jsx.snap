// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should render correctly 1`] = `
<div>
  <h1>
    Front-End ToolKit
  </h1>
</div>
`;
